primitives/chains/westend/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges Common.
+
+// Parity Bridges Common is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Bridges Common is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+// RuntimeApi generated functions
+#![allow(clippy::too_many_arguments)]
+// Runtime-generated DecodeLimit::decode_all_with_depth_limit
+#![allow(clippy::unnecessary_mut_passed)]
+
+use bp_message_lane::{LaneId, Weight, MessageNonce};
+use sp_std::prelude::*;
+
+pub use bp_polkadot_core::*;
+
+/// Westend Chain
+pub type Westend = PolkadotLike;
+
+/// Name of the `WestendHeaderApi::best_blocks` runtime method.
+pub const BEST_WESTEND_BLOCKS_METHOD: &str = "WestendHeaderApi_best_blocks";
+/// Name of the `WestendHeaderApi::finalized_block` runtime method.
+pub const FINALIZED_WESTEND_BLOCK_METHOD: &str = "WestendHeaderApi_finalized_block";
+/// Name of the `WestendHeaderApi::is_known_block` runtime method.
+pub const IS_KNOWN_WESTEND_BLOCK_METHOD: &str = "WestendHeaderApi_is_known_block";
+/// Name of the `WestendHeaderApi::incomplete_headers` runtime method.
+pub const INCOMPLETE_WESTEND_HEADERS_METHOD: &str = "WestendHeaderApi_incomplete_headers";
+
+sp_api::decl_runtime_apis! {
+	/// API for querying information about Westend headers from the Bridge Pallet instance.
+	///
+	/// This API is implemented by runtimes that are bridging with Westend chain, not the
+	/// Westend runtime itself.
+	pub trait WestendHeaderApi {
```

Comment:
These APIs (and associated constants above) are now obsolete actually - there should be something like `WestendFinalityApi`. Or are you just trying to make something compilable?